units/halt.target

```diff
@@ -11,9 +11,13 @@
 Description=System Halt
 Documentation=man:systemd.special(7)
 DefaultDependencies=no
+Requires=shutdown.target umount.target final.target
+After=shutdown.target umount.target final.target
 Requires=systemd-halt.service
-After=systemd-halt.service
+Before=systemd-halt.service
 AllowIsolate=yes
+JobTimeoutSec=30min
+JobTimeoutAction=halt-force
```

Comment:
Could you split out this part of the change?

Reply:
-> #32902.